index.html

```diff
@@ -0,0 +1,880 @@
+              <em>wrote</em>
+            </div>
+            <div class="artile-author-rol">
+              <h1>
+                Creating Secure Password Flows With NodeJS And MySQL
+                <span> <i class="fas fa-comment-alt"></i>Leave a comment </span>
+              </h1>
+            </div>
+            <div class="article-author-tech">
+              <span> <i>#</i> Node.js <sup>32</sup> </span>
+              <span> <i>#</i>JavaScript <sup>253</sup> </span>
+              <span> <i>#</i>Security <sup>47</sup> </span>
+            </div>
+            <div class="article-author-arrow"><i> →</i></div>
+          </div>
+        </div>
+        <div class="author-card-box-3">
+          <div class="header-article-wrapper">
+            <div class="article-author-title">
+              <div class="image-wrapper">
+                <img
+                  src="https://cloud.netlifyusercontent.com/assets/344dbf88-fdf9-42bb-adb4-46f01eedd629/f40da66c-f8fa-4fa1-8f57-607fc8e676ee/darshan-bio-200x200.jpg"
+                  alt="Darshan Somashekar"
+                />
+              </div>
+              <h1>Darshan Somashekar</h1>
+              <em>wrote</em>
+            </div>
+            <div class="artile-author-rol">
+              <h1>
+                Creating Secure Password Flows With NodeJS And MySQL
+                <span> <i class="fas fa-comment-alt"></i>Leave a comment </span>
+              </h1>
+            </div>
+            <div class="article-author-tech">
+              <span> <i>#</i> Node.js <sup>32</sup> </span>
+              <span> <i>#</i>JavaScript <sup>253</sup> </span>
+              <span> <i>#</i>Security <sup>47</sup> </span>
+            </div>
+            <div class="article-author-arrow"><i> →</i></div>
+          </div>
+        </div>
+        <div class="author-card-box-4">
+          <div class="header-article-wrapper">
+            <div class="article-author-title">
+              <div class="image-wrapper">
+                <img
+                  src="https://cloud.netlifyusercontent.com/assets/344dbf88-fdf9-42bb-adb4-46f01eedd629/f40da66c-f8fa-4fa1-8f57-607fc8e676ee/darshan-bio-200x200.jpg"
+                  alt="Darshan Somashekar"
+                />
+              </div>
+              <h1>Darshan Somashekar</h1>
+              <em>wrote</em>
+            </div>
+            <div class="artile-author-rol">
+              <h1>
+                Creating Secure Password Flows With NodeJS And MySQL
+                <span> <i class="fas fa-comment-alt"></i>Leave a comment </span>
+              </h1>
+            </div>
+            <div class="article-author-tech">
+              <span> <i>#</i> Node.js <sup>32</sup> </span>
+              <span> <i>#</i>JavaScript <sup>253</sup> </span>
+              <span> <i>#</i>Security <sup>47</sup> </span>
+            </div>
+            <div class="article-author-arrow"><i> →</i></div>
+          </div>
+        </div>
+      </section>
```

Comment:
You can include the titles in these four articles, but you have to change the bg color to a gray scale, depending on the hierarchy. 

Also, use the `<article>` tag instead of the div one.